_commands/tools/bzip2.md

```diff
@@ -0,0 +1,44 @@
+---
+---
+
+bzip2
+-------
+<!--TODO: Add documentation for this command by submitting a pull request.-->
+<!-- one line explanation would go here -->
+`bzip2` is the command used for compressing and decompressing files.
+
+<!-- minimal example -->
+~~~ bash
+$ bzip2 filename
+~~~
+
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `bzip2`
+
+~~~bash
+$ bzip2 [Option(s)] filename(s)
+~~~
+
+Options include:
```

Comment:
Are these the only useful flags? Consider going through this list to check 
 if other flags may be useful: https://www.tutorialspoint.com/unix_commands/bzip2.htm (for example, `-s`. `--fast`, etc. may be quite useful)

Reply:
more useful flags added